sdk/core/azure-mgmt-core/setup.py

```diff
@@ -49,11 +49,10 @@
         'Programming Language :: Python',
         'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```

Comment:
add in 3.11 or is that a separate PR ?

Reply:
I will let mgmt team decide if they are ready to add 3.11. :)